### PR TITLE
Created CardContainer abstract class

### PR DESCRIPTION
**Pull request by gustavoSo3**
This class is the base for:

* Deck
* Hand
* Board

There is no way to test this class at the moment but once any of the previously mentioned classes are created we can fully test the functionality

### CODE REVIEW

File: frontend/src/card_container/card_container.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        self._cards: List[Card] = list()
+
+    @property
+    def is_empty(self) -> bool:
+        """Is the card container empty"""
+        return len(self._cards) == 0
+
+    def get_all(self) -> List[Card]:
+        """Returns a copy the current list of cards of this container"""
+        return list(self._cards)
```
Comment (BamCurti) [inline frontend/src/card_container/card_container.py]: The ticket specifies the reference of the list must be deleted from container.

**Review by BamCurti — CHANGES_REQUESTED** [review]
It almost finish, just delete de reference of the list of cards when get_all method is called

File: frontend/src/card_container/card_container.py (hunk 1 above)
Comment (gustavoSo3) [reply frontend/src/card_container/card_container.py]: Oh yeah my bad xD.

**Review by BamCurti — APPROVED** [review]
thanks!

**Review by BamCurti — APPROVED** [review]

**Pull request merged**